libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -120,19 +118,11 @@ private const val MAPBOX_NAVIGATION_TOKEN_EXCEPTION = "You need to provide an ac
  *
  * @param navigationOptions a set of [NavigationOptions] used to customize various features of the SDK.
  * Use [defaultNavigationOptionsBuilder] to set default options
- * @param locationEngine used to listen for raw location updates
- * @param locationEngineRequest used to request raw location updates
  */
 class MapboxNavigation
-@JvmOverloads
 constructor(
```

Comment:
We can remove `constructor`

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/3253